src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
+    protected String from;
+    protected String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + this.from + " to: " + this.to + ")";
+
```

Comment:
Trailing whitespaces are quite prevalent in your code. Just remove them and you're good.